### PR TITLE
GSW-764 Refactor: page-views-layout

### PR DESCRIPTION
**Pull request by AkydA**
GSW-764

Description
---
- Refactor page, views, layout directory structures
- `EarnAdd` component is merged into `PoolAdd`

### CODE REVIEW

File: packages/web/pages/earn/pool/stake.tsx [hunk 1]
```diff
@@ -1,21 +1,14 @@
...
+import { DEFAULT_I18N_NS, SEOInfo } from "@constants/common.constant";
 import { SwapFeeTierInfoMap } from "@constants/option.constant";
+import useRouter from "@hooks/common/use-custom-router";
+import { useGnotToGnot } from "@hooks/token/use-gnot-wugnot";
+import PoolStake from "@views/pool/pool-stake/PoolStake";
 import { makeSwapFeeTier } from "@utils/swap-utils";
-import { DEFAULT_I18N_NS, SEOInfo } from "@constants/common.constant";
-import { serverSideTranslations } from "next-i18next/serverSideTranslations";
-import { useTranslation } from "react-i18next";
+import { useGetPoolDetailByPath } from "src/react-query/pools";
```
Comment (jinoosss) [inline packages/web/pages/earn/pool/stake.tsx:11]: change path to `@query/*`.

File: packages/web/src/views/pool/pool-add/PoolAdd.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  const { width } = useWindowSize();
+  const router = useCustomRouter();
+  const poolPath = router.getPoolPath();
+  const { data, isLoading } = useGetPoolDetailByPath(poolPath, {
+    enabled: !!poolPath,
+  });
+  const { getGnotPath } = useGnotToGnot();
+  const { isLoading: isLoadingCommon } = useLoading();
+
+  const hasDedicatedPool = router.asPath.includes("/pool");
```
Comment (jinoosss) [inline packages/web/src/views/pool/pool-add/PoolAdd.tsx:33]: This part is explicitly taken as component props.

**Review by jinoosss — APPROVED** [review]
Nice work !
I've left a few comments, but the behavior is fine, and it seems like a good idea to create a separate PR to work with since there are a lot of file modifications.

Also, modularization by directory would simplify the import path.

**Review by roje1 — APPROVED** [review]
LGTM 👍

File: packages/web/src/views/pool/pool-add/PoolAdd.tsx (hunk 2 above)
Comment (AkydA) [reply packages/web/src/views/pool/pool-add/PoolAdd.tsx:33]: > This part is explicitly taken as component props.

Will do in next refactoring soon!

**Pull request merged**